CONTRIBUTING.rst

```diff
@@ -135,9 +135,9 @@ Before you submit a pull request, check that it meets these guidelines:
    Github Actions.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
-   feature to the list in README.md.
-3. The pull request should work for Python 3.7, or later. Check
-   https://github.com/elfi-dev/elfi/actions/workflows/pytest.yml
+   feature to the list in README.rst.
+3. The pull request should work for Python 3.5 and later. Check
+   https://travis-ci.org/elfi-dev/elfi/pull_requests
```

Comment:
i think the link in our current version is correct: https://github.com/elfi-dev/elfi/actions/workflows/pytest.yml